src/drivers/intel/dw-dma.c

```diff
@@ -1190,8 +1200,10 @@ static void dw_dma_irq_handler(void *data)
 			}
 
 			p->chan[i].lli_current->ctrl_hi &= ~DW_CTLH_DONE(1);
-			dcache_writeback_region(p->chan[i].lli_current,
-						sizeof(*p->chan[i].lli_current));
+
+			size_t len = sizeof(*p->chan[i].lli_current);
```

Comment:
Ok. I don't like it because I have to declare unnecessary variable in function scope just to satisfy checkpatch (80 chars limit), so I wanted to limit scope at least, but if it has to be on top then it will be ;)